From 7c923f1de89f80b511338f9b434acfdf2547dbac Mon Sep 17 00:00:00 2001
From: Jonathan Lebon <jonathan@jlebon.com>
Date: Wed, 23 Sep 2020 16:18:18 -0400
Subject: [PATCH] 00systemd: add missing cryptsetup-related targets

We want these in the initramfs. Things related to clevis and systemd's
`cryptsetup-generator` reference these targets.
---
 modules.d/00systemd/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index 17be74fd..bbce31a2 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -65,6 +65,8 @@ install() {
         $systemdutildir/system-generators/systemd-gpt-auto-generator \
         \
         $systemdsystemunitdir/cryptsetup.target \
+        $systemdsystemunitdir/cryptsetup-pre.target \
+        $systemdsystemunitdir/remote-cryptsetup.target \
         $systemdsystemunitdir/emergency.target \
         $systemdsystemunitdir/sysinit.target \
         $systemdsystemunitdir/basic.target \

